Trading/Mode/daily_trading_mode/daily_trading.py

```diff
@@ -525,6 +525,11 @@ async def _create_order(
             oco_group = self.exchange_manager.exchange_personal_data.orders_manager.create_group(group_type)
             for order in chained_orders:
                 order.add_to_order_group(oco_group)
+            if self.exchange_manager.trader.enable_inactive_orders:
+                active_order_swap_strategy = oco_group.get_active_order_swap_strategy(0)
```

Comment:
this method will like change to add more flexibility in how inactive order strategies are chosen instead of it being static to an order group